docs/examples/metrics/example.py

```diff
@@ -17,15 +18,17 @@
 set_meter_provider(provider)
 
 
-def observable_counter_func() -> Iterable[Observation]:
+def observable_counter_func(options: CallbackOptions) -> Iterable[Observation]:
     yield Observation(1, {})
 
 
-def observable_up_down_counter_func() -> Iterable[Observation]:
+def observable_up_down_counter_func(
+    options: CallbackOptions,
```

Comment:
Just some random thought in my mind. This is probably out of scope, Should there be a way for users to pass the extra arguments for callback. They could probably achieve this by creating a partial function and writing a wrapper around it.

Reply:
@srikanthccv can we resolve this conversation?

Reply:
resolved